demo_example/CMakeLists.txt

```diff
@@ -32,3 +32,18 @@ target_link_libraries(http_client async_simple)
 
 add_executable(block_http_server http/block_http/block_http_server.cpp)
 target_link_libraries(block_http_server async_simple)
+
+SET(ENABLE_SSL OFF)
```

Comment:
Why this is disabled by default?

Reply:
I think many people don't install openssl and i hope the client can be used immediately, no other dependency as default.